doma-processor/src/main/java/org/seasar/doma/internal/apt/annot/BatchModifyAnnot.java

```diff
@@ -95,6 +101,14 @@ public AnnotationValue getIgnoreGeneratedKeys() {
     return ignoreGeneratedKeys;
   }
 
+  public DuplicateKeyType getDuplicateKeyType() {
```

Comment:
We have a rule to add the `Value` suffix to method names that get resolved values of `AnnotationValue`.
Therefore, please change the name to `getDuplicateKeyValue`.